test/Microsoft.AspNetCore.Mvc.IntegrationTests/ModelBindingTestContext.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.IntegrationTests
+{
+    public class ModelBindingTestContext : ControllerContext
```

Comment:
Like it! Was getting concerned about reanimating `OperationBindingContext` as a test concept. But this is a nice hedge to reduce test code churn.